client/public/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>countdown</title>
+</head>
+<body>
+  <div id="countdown"></div>
+  <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
+  <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
+  <h1>Review</h1>
+</body>
+<script type="text/javascript" src="./client/public/bundle.js"></script>
+</html>
```

Comment:
Needs newline

Reply:
What do you mean by the newline comments? Thanks!

Reply:
Git gives an error message at the end of files if they don't end in a new line - https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline